site/app/controllers/forum/ForumController.php

```diff
@@ -369,6 +369,38 @@ public function publishThread() {
         return $this->core->getOutput()->renderJsonSuccess($result);
     }
 
+    /**
+     * @Route("/courses/{_semester}/{_course}/forum/make_announcement", methods={"POST"})
+     * @AccessControl(permission="forum.modify_announcement")
```

Comment:
If only Full Access Graders and higher can call the other function why not just make the same here?

Reply:
I used forum.modify_announcement just to follow the overall trend of the rest of the file, there was no specific reason for using that over full_access_grader. Looking at access.php, it doesn't seem like there was much of a difference between the two.